tjreports/site/views/reports/view.html.php

```diff
@@ -39,8 +39,11 @@ class TjreportsViewReports extends JViewLegacy
 	public function display($tpl = null)
 	{
 		$canDo = TjreportsHelpersTjreports::getActions();
-		$user = JFactory::getUser();
-		$user_id = $user->id;
+		$this->user       = JFactory::getUser();
+		$this->user_id    = $this->user->id;
+
+		$user = $this->user;
+		$user_id = $this->user_id;
```

Comment:
@hemantpurohit1 use $this->user and $this->user_id

remove
$user = $this->user;
$user_id = $this->user_id;